ubiquibot-config-default.json

```diff
@@ -93,37 +93,12 @@
       "enabled": false
     }
   ],
-  "comment-element-pricing": {
-    "blockquote": 0,
-    "break": 0,
-    "code": 0,
-    "definition": 0,
-    "emphasis": 0,
-    "heading": 0,
-    "html": 0,
-    "image": 0,
-    "imageReference": 0,
-    "inlineCode": 0,
-    "link": 0,
-    "linkReference": 0,
-    "list": 0,
-    "listItem": 0,
-    "paragraph": 0,
-    "root": 0,
-    "strong": 0,
-    "text": 0,
-    "thematicBreak": 0,
-    "delete": 0,
-    "footnoteDefinition": 0,
-    "footnoteReference": 0,
-    "table": 0,
-    "tableCell": 0,
-    "tableRow": 0,
-    "alignType": 0,
-    "flowContent": 0,
-    "listContent": 0,
-    "phrasingContent": 0,
-    "rowContent": 0,
-    "tableContent": 0
```

Comment:
Are you able to restore these in HTML tag version?

Reply:
I'm not sure what you mean. They are all zeroes so it's same as if they are not even in the config.

Reply:
I understood that the preference inherit (somewhat like CSS) from the `ubiquibot-default.json` and then override with the more local config (like this repository's `.github/ubiquibot-config.json`) UNLESS the property is specified as having a value of `0` for example.

Reply:
We only have inheritance on top level properties, so if `incentives` property is missing from repo and org config, it will use the whole `incentives` property from default config. Nested properties don't have inheritance, that is something I was meaning to make an issue for.

Reply:
Please do. You can use the context menu of your comment to back link to it for context for the bounty hunter.